users/serializers.py

```
@@ -0,0 +1,61 @@
+    email = serializers.EmailField()
+    password = serializers.CharField(write_only=True)
+
+    def validate(self, data):
+        email = data.get('email')
+        password = data.get('password')
+
+        if email and password:
+            # backends.py 설정을 통해 이메일 인증 진행행
+            user = authenticate(username=email, password=password)
+            if not user:
+                raise serializers.ValidationError("이메일 또는 비밀번호가 틀렸습니다.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)